source/manage/command-line-tools.rst

```diff
@@ -111,2701 +103,341 @@ Options
 
 Description
-  List all channels on a specified team. Private channels are appended with ``(private)`` and archived channels are appended with ``(archived)``.
+  Resets the database to its initial state. Doesn't start the application server. Only starts the store layer and truncates the tables, excluding the ``migrations`` table.
 
 Format
   .. code-block:: none
 
-    mattermost channel list {teams}
+    mattermost db reset
 
 Example
   .. code-block:: none
 
-    bin/mattermost channel list myteam
+    bin/mattermost db reset
 
-mattermost channel modify
-~~~~~~~~~~~~~~~~~~~~~~~~~
+mattermost db version
+~~~~~~~~~~~~~~~~~~~~~~
 
 Description
-  Modify a channel's public/private type.
+  Returns the most recently applied version number.
 
 Format
   .. code-block:: none
 
-    mattermost channel modify
+    mattermost db version
 
 Example
   .. code-block:: none
 
-    bin/mattermost channel modify myteam:mychannel --username myusername --private
+    bin/mattermost export actiance --exportFrom=1513102632
 
 Options
   .. code-block:: none
 
-    --username [REQUIRED] Username of the user who is changing the channel privacy.
-    --public   Change a private channel to be public.
-    --private  Change a public channel to be private.
-
-mattermost channel move
-~~~~~~~~~~~~~~~~~~~~~~~
-
-.. note::
-
-  From Mattermost v6.0, this command has been replaced with the mmctl command `mmctl channel move </manage/mmctl-command-line-tool.html#mmctl-channel-move>`__.
-
-Description
-  Move channels to another team. The command validates that all users in the channel belong to the target team. Incoming/outgoing webhooks are moved along with the channel. Channels can be specified by ``[team]:[channel]`` or by using channel IDs. Channel IDs can be obtained via the `API <https://api.mattermost.com/#operation/GetChannelByName>`_ or the `mattermost channel search </manage/mmctl-command-line-tool.html#mmctl-channel-search>`__ command.
-    
-Format
-  .. code-block:: none
-
-    mattermost channel move
+    --exportFrom string     Unix timestamp (milliseconds since epoch, UTC) to export data from.
+    --batchSize int         The number of posts to export. The default of -1 means no limit.
 
-Example
-  .. code-block:: none
 
-    bin/mattermost channel move newteam 8soyabwthjnf9qibfztje5a36h --username myusername
-    bin/mattermost channel move newteam myteam:mychannel --username myusername
+----
 
-Options
-  .. code-block:: none
+mattermost export
+-----------------
 
-    --username [REQUIRED]                 Username of the user who is moving the team.
-    --remove-deactivated-users [OPTIONAL] When moving the channel, remove any users who have been deactivated who may be preventing the move.
+.. include:: ../_static/badges/ent-only.rst
+  :start-after: :nosearch:
 
-mattermost channel remove
-~~~~~~~~~~~~~~~~~~~~~~~~~
+Description
+  Commands for exporting data for compliance and for merging multiple Mattermost instances.
 
-.. note::
+Child Commands
```

Comment:
Are these export options supported by `mmctl`? Seems an omission we should fix if not.